Wrapping/R/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ set_source_files_properties(${swig_generated_file_fullname} PROPERTIES COMPILE_F
 # set the output directory for the R library to the binary packaging location
 set_target_properties( ${SWIG_MODULE_SimpleITK_TARGET_NAME} PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Packaging/SimpleITK/src/)
 
+if (WIN32)
+	set_target_properties( ${SWIG_MODULE_SimpleITK_TARGET_NAME} PROPERTIES LINK_FLAGS "-s -shared -static-libgcc -static-libstdc++ -static -lpthread" )
+endif()
```

Comment:
I'll look at these closer. Why are these flags needed? Is it because the recommended mingw, version is going to have incompatible versions of the libraries with the R run-time. This looks like some pretty sketchy stuff. I can see having different versions of stdc++, but mixing different version of pthread, and libgcc seems like it will cause problems sooner or later.

Does setting the CMAKE_*_LINKER_FLAGS to these values work?